shell/ptty_windows.go

```diff
@@ -11,6 +13,25 @@ import (
 	"github.com/gruntwork-io/terragrunt/options"
 )
 
+func PrepareConsole() {
+	enableVirtualTerminalProcessing(os.Stderr)
+	enableVirtualTerminalProcessing(os.Stdout)
```

Comment:
Do we need to do it [for stdin too](https://github.com/containerd/console/blob/f652dc3/console_windows.go#L33-L44)?

Reply:
I was thinking about it to be honest and I don't know, because we don't input any escaped characters. But I guess I'll add it for completeness sake.